README.md

```diff
@@ -455,6 +455,8 @@ const chatAdapter = await chatSDK.createChatAdapter();
 
 > :warning: Currently supported on web only
 
+> :warning: Please ensure voice & video call is stopped before leveraging endChat SDK method
```

Comment:
Would it be more clear if the following was written instead “`VoiceVideoCallingSDK.stopCall()` must be called before `chatSDK.endChat()`”?

Users would know better which method to be called.